clarity/src/vm/analysis/mod.rs

```diff
@@ -141,6 +141,7 @@ pub fn run_analysis(
             | StacksEpochId::Epoch22
             | StacksEpochId::Epoch23
             | StacksEpochId::Epoch24
+            | StacksEpochId::Epoch25
```

Comment:
Epoch 2.5 is only used to deploy `pox-4`, which must happen one reward cycle prior to epoch 3.0.  This is because Nakamoto's PoX contract needs different functions than those in `pox-3`, and there need to be stackers ready to go when the first epoch 3.0 Bitcoin block gets mined (meaning, there must be stackers who stacked in `pox-4`).